bilby/gw/detector/calibration.py

```diff
@@ -12,9 +12,15 @@
 from ..prior import CalibrationPriorDict
 
 
-def read_calibration_file(filename, frequency_array, number_of_response_curves, starting_index=0):
-    """
-    Function to read the hdf5 files from the calibration group containing the physical calibration response curves.
+def read_calibration_file(filename, frequency_array, number_of_response_curves, starting_index=0, correction=None):
```

Comment:
This is a minor point, but can the new parameter be called `correction_type` rather than `correction`? The latter to me seems like an optional boolean variable, like whether to implement a higher-order correction.